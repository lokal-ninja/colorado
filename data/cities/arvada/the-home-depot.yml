id: 210073922
type: node
latitude: 39.79309
longitude: -105.0835
postcode: "80002"
city: Arvada
street: Wadsworth Boulevard
housenumber: "5215"
phone: +1 303-456-4000
opening_hours: Mo-Sa 06:00-22:00; Su 07:00-20:00
website: https://www.homedepot.com/l/Arvada/CO/Arvada/80002/1502
