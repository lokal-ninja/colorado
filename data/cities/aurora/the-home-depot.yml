id: 2854537129
type: node
shop: doityourself
latitude: 39.6986398
longitude: -104.82476120000001
postcode: "80012"
city: Aurora
street: "East Mississippi Avenue"
housenumber: 14001
phone: "303-743-7557"
opening_hours: ""
website: "https://www.homedepot.com/"
