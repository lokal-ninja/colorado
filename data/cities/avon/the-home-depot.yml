id: 239077705
type: way
latitude: 39.6259
longitude: -106.4979
postcode: "81620"
city: Avon
street: "Yoder Avenue"
housenumber: 295
phone: "+1 970-748-9483"
opening_hours: "Mo-Sa 07:00-21:00; Su 07:00-20:00"
website: "https://www.homedepot.com/l/avon-vail/co/avon/81620/1525"