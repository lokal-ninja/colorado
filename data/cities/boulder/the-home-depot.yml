id: 98564034
type: node
latitude: 40.01624
longitude: -105.25571
postcode: "80301"
city: Boulder
street: 29th Street
housenumber: "1600"
phone: +1 303-449-4221
opening_hours: Mo-Sa 06:00-22:00; Su 07:00-20:00
website: https://www.homedepot.com/l/Boulder/CO/Boulder/80301/1546
