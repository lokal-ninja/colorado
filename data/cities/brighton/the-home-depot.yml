id: 220893020
type: node
latitude: 39.95212
longitude: -104.78778
postcode: "80601"
city: Brighton
street: Buckley Road
housenumber: "2440"
phone: +1 303-655-1686
opening_hours: Mo-Sa 06:00-22:00; Su 07:00-20:00
website: https://www.homedepot.com/l/brighton-co/co/brighton/80601/1547
