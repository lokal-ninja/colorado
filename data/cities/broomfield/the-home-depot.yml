id: 132725544
type: way
shop: doityourself
latitude: 39.9166043
longitude: -105.0579527
postcode: "80020"
city: Broomfield
street: "Sheridan Boulevard"
housenumber: 12171
phone: ""
opening_hours: "Mo-Sa 06:00-21:00; Su 07:00-20:00"
website: ""
