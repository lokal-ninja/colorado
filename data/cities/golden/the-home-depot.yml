id: 92735943
type: way
shop: doityourself
latitude: 39.7221202
longitude: -105.1908128
postcode: "80401"
city: Golden
street: ""
housenumber: 
phone: ""
opening_hours: ""
website: ""
